docs/extend/authorization.md

```diff
@@ -249,4 +251,5 @@ Response body           | []byte            | Raw docker daemon response body
 Name    | Type   | Description
 --------|--------|----------------------------------------------------------------------------------
 Allow   | bool   | Boolean value indicating whether the response is allowed or denied
-Message | string | Authorization message (will be returned to the client in case the access is denied)
\ No newline at end of file
+Message | string | Authorization message (will be returned to the client in case the access is denied)
+Error   | string | Error message (will be returned to the client in case the plugin encounter an error)
```

Comment:
Hm, yes, they look fairly inconsistent :/

Are you okay with just changing `Err` then for now? We can do the others in a follow-up